esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -598,10 +609,12 @@ define(
     Replicate: "Replicar",
     ReplicatedLost: "Réplica perdida",
     ReplicateOffset: "Offset de Reproducción",
+    Report: "Reporte",
     ReportAnError: "Reportar un error",
     ReportError: "Informe de error",
     RepresentsASubset: "Representar un subconjunto de todas los casamientos. Usando un filtro correcto puede reducir el numero de casamientos",
     RequestSchema: "Esquema del requerimiento",
+    RequiredForFixedSpray: "Obligatorio para spray fijo ",
```

Comment:
Extra space at the end of line.... but also, what's the context for this message, it just sounds odd as is...

Reply:
Changes made and trailing spaces removed